.github/workflows/release.yaml

```diff
@@ -205,3 +207,17 @@ jobs:
       run: gh release upload ${{ needs.release.outputs.release-tag }} ${{ needs.provenance.outputs.provenance-name }}
       env:
         GH_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+
+  # Send out release notifications after the Release was published on GitHub.
+  notifications:
+    needs: [release]
+    name: Send Release notifications
+    uses: ./.github/workflows/_release-notifications.yaml
+    permissions:
+      contents: read
+    with:
+      repo_name: ${{ github.event.repository.name }}
+      release_tag: ${{ needs.release.outputs.release-tag }}
+      release_url: ${{ needs.release.outputs.release-url }}
+    secrets:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
The original Release Notification was disabled:

https://github.com/jenstroeger/python-package-template/blob/7bf474b781efa1fb0e69dcf31f968675695f10bc/.github/workflows/release-notifications.yaml#L15-L17

This one is not because without the required [Slack Webhook URL](https://api.slack.com/messaging/webhooks) secret it won’t run anyway. (Can’t find docs anymore that said that Actions without sufficient secrets don’t run.)

Reply:
> This one is not because without the required [Slack Webhook URL](https://api.slack.com/messaging/webhooks) secret it won’t run anyway. (Can’t find docs anymore that said that Actions without sufficient secrets don’t run.)

I think it's better to be explicit and add the condition back. Also it would be good to add a note to README.md that if secrets are not set, the workflow might not work at all.

Reply:
Makes sense, commit https://github.com/jenstroeger/python-package-template/pull/398/commits/e70c60770d2b7bb4951286f03c0a3d258bf9c788.